server/generate.py

```diff
@@ -118,8 +118,8 @@ def gen_assignment(course):
     name = course.offering + '/' + display_name.lower().replace(' ', '')
 
     last_night = (datetime.datetime.utcnow()
-                          .replace(hour=0, minute=0, second=0, microsecond=0)
-                          - datetime.timedelta(seconds=1))
```

Comment:
Makes sense. Interestingly Flake8 threw a W503 at me when the line break was before the operator. Seems like that's flake8/pycodestyle hasn't been updated to the latest pep8 change 

https://gitlab.com/pycqa/flake8/issues/139